include/openssl/ec.h

```diff
@@ -248,6 +248,17 @@ const EC_POINT *EC_GROUP_get0_generator(const EC_GROUP *group);
 */
 BN_MONT_CTX *EC_GROUP_get_mont_data(const EC_GROUP *group);
 
+/** Perfroms inversion modulo order of a EC_GROUP
+ *  \param group  EC_GROUP object
+ *  \param res    BIGNUM to which the result is copied
+ *  \param x      BIGNUM of which inverse to compute
+ *  \param ctx    BN_CTX object
+ *  \param constantTime int   if 0 - not constant time
+ *  \return 1 on success and 0 if an error occurred
+*/
+int EC_GROUP_do_inverse_ord(const EC_GROUP *group, BIGNUM *res,
+                            const BIGNUM *x, BN_CTX *ctx, int constantTime);
```

Comment:
Is constantTime = 0 worth it? I'd much prefer not to even have this option, esp exposed in the public API.